src/scripts/count-references-to-contributor-docs/cli.ts

```diff
@@ -15,7 +15,7 @@ const REPOSITORY_NAMES = [
   'snaps',
 ] as const;
 
-type RepositoryName = typeof REPOSITORY_NAMES[number];
+type RepositoryName = (typeof REPOSITORY_NAMES)[number];
```

Comment:
It’s not related to the newly added workflow, but the lint workflow was failing.
```
Error:   18:23  error  Replace `typeof·REPOSITORY_NAMES` with `(typeof·REPOSITORY_NAMES)`  prettier/prettier